packages/create-sitecore-jss/src/templates/angular/angular.json

```diff
@@ -115,9 +118,10 @@
         "server": {
           "builder": "@angular-devkit/build-angular:server",
           "options": {
-            "outputPath": "dist/server",
             "main": "server.bundle.ts",
-            "tsConfig": "src/tsconfig.server.json"
+            "tsConfig": "src/tsconfig.server.json",
+            "deleteOutputPath": false,
```

Comment:
It deletes _outputPath_ on _ng build_
The problem here, that when it's triggered it removes also _browser_ bundle from the output path, which is not needed
(because we output server bundle under /dist, instead of /dist/server now). Server build is triggered after the browser build.
Even if the server bundle is not removed and there is another build - it will just override existing files